packages/utilities/src/customizable.tsx

```diff
@@ -1,17 +1,21 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { GlobalSettings, IChangeDescription } from './GlobalSettings';
+import { Customizations } from './Customizations';
+
+export function customizable(
+  scope: string,
+  fields: string[]
+): <P, S>(ComposedComponent: new (props: P, ...args: any[]) => React.Component<P, S>) => any {
 
-export function customizable(fields: string[]): <P, S>(ComposedComponent: new (props: P, ...args: any[]) => React.Component<P, S>) => any {
   // tslint:disable-next-line:no-shadowed-variable
   return function customizableFactory<P, S>(
     ComposedComponent: (new (props: P, ...args: any[]) => React.Component<P, S>)
   ): any {
     return class ComponentWithInjectedProps extends React.Component<P, {}> {
       public static contextTypes: {
-        injectedProps: PropTypes.Requireable<any>;
+        customizations: PropTypes.Requireable<any>;
       } = {
-        injectedProps: PropTypes.object
+        customizations: PropTypes.object
```

Comment:
Forgive my ignorance... Why are you using `PropTypes` here?

Reply:
This is how React context knows what context values to provide the component.

See here:
https://facebook.github.io/react/docs/context.html

Reply:
Right right, I know about those. My question was, why include them when you're in a TypeScript codebase?

I've noticed that now, as a consumer, I have to install `@types/prop-types` because `Customizer` depends on them, but nowhere else in my codebase I'm using `prop-types`, so I wouldn't really need them.

Reply:
Aaha, understood.